pkg/bugtool/bugtool.go

```diff
@@ -83,6 +84,14 @@ func doSaveInitInfo(fname string, info *InitInfo) error {
 		logger.GetLogger().WithField("bpftool", info.BpfToolPath).Info("Successfully detected bpftool path")
 	}
 
+	gops, err := exec.LookPath("gops")
```

Comment:
Why do we need to do this on the agent?

Reply:
cause IIRC the agent has the path of installation in systemd , like bpftool so it finds the path for later

Reply:
So, we get the path in the agent we save it into the tetragon json inside /run/tetragon/, and later tetra tries to auto read it allowing users to get dump without specifying the search path or the full gops path.